src/components/ifxToggle/ifxToggle.tsx

```diff
@@ -0,0 +1,28 @@
+// pill-toggle.tsx
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-toggle',
+  styleUrl: 'ifxToggle.scss',
+  shadow: true,
+})
+export class ifxToggle {
+  @Prop({ mutable: true }) checked: boolean = false;
+
+  @Event({ eventName: 'valueChanged' }) valueChanged: EventEmitter<boolean>;
+
+  toggle() {
+    this.checked = !this.checked;
+    this.valueChanged.emit(this.checked);
+
+    console.log(this.checked);
+  }
+
+  render() {
+    return (
+      <div class={`pill-container ${this.checked ? 'checked' : ''}`} onClick={() => this.toggle()}>
+        <div class={`pill-switch ${this.checked ? 'checked' : ''}`} />
+      </div>
+    );
```

Comment:
In the Figma design of the `toggle`, the user should be allowed to add `label` inside two sides of a toggle.

Reply:
Switch =/= Toggle
<img width="266" alt="toggle" src="https://user-images.githubusercontent.com/108734092/233096291-67c97bbd-4a88-4ced-a2d7-e602736d562f.PNG">